src/main/java/formflow/library/data/SubmissionRepositoryService.java

```diff
@@ -116,32 +116,38 @@ public void removeSubflowCSRF(Submission submission, String subflowName) {
      */
     public synchronized void generateAndSetUniqueShortCode(Submission submission) {
 
+        if (shortCodeConfig.getConfig(submission.getFlow()) == null) {
+            return;
+        }
+
         if (submission.getShortCode() != null) {
             log.warn("Unable to create short code for submission {}", submission.getId());
```

Comment:
Is this because one would already exist? If so it might help to make the log a little clearer by saying so.

Reply:
Yep, but i agree that the warn message should just more clear.